src/main/java/seedu/address/model/application/PositionContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.model.application;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+
+/**
+ * Tests that an {@code Application}'s {@code Position} matches any of the keywords given.
+ */
+public class PositionContainsKeywordsPredicate implements Predicate<Application> {
```

Comment:
Technically I think we don't need this class for now since we aren't doing find for v1.2. But this would be nice for next versions if we implement to find command, so we can just keep first (and maybe add one for Company as well later on?)

Reply:
Yup, already added the Company one for this